### PR TITLE
Update Cloud Shell contact

### PR DESCRIPTION
**Pull request by mccoyp**
# Description

I reached out to Danny because our [nightly pipeline](https://dev.azure.com/azure-sdk/public/_build/results?buildId=3953681&view=logs&j=a460d732-23aa-5493-a411-cc406067acf8&t=8c98ab76-57b2-59d0-3d3a-18dce788f3ba) complained about private Azure membership, but was told that Janice D'Sa (@dsajanice) would be a better contact. Janice is already a public Azure organization member.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by mccoyp** [comment]
@weshaggard based on the [pipeline error](https://dev.azure.com/azure-sdk/public/_build/results?buildId=3954289&view=logs&j=a460d732-23aa-5493-a411-cc406067acf8&t=2b56d21f-1cfb-5a3c-ae52-6b8b1a9ee09a) -- and the fact that Janice is already a [public member of Azure](https://github.com/orgs/Azure/teams?query=%40dsajanice) -- it looks like write permissions might be the issue. Do you know how to resolve this?

**Comment by weshaggard** [comment]
Correct https://github.com/orgs/Azure/teams/azure-sdk-partners?query=membership%3Achild-team+dsajanice+

They need to request permission, see https://aka.ms/azsdk/access

**Comment by mccoyp** [comment]
Janice suggested that we reach out to @mbifeld, and Mitchell requested access but will need to wait for his manager to approve after time off. Once that's sorted out, I can update this PR to list Mitchell as our contact.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**
